pxr/imaging/garch/CMakeLists.txt

```diff
@@ -1,17 +1,24 @@
 set(PXR_PREFIX pxr/imaging)
 set(PXR_PACKAGE garch)
 
-if (NOT ${PXR_ENABLE_GL_SUPPORT})
+if ((NOT ${PXR_ENABLE_GL_SUPPORT}) AND (NOT APPLEIOS))
     message(STATUS
-        "Skipping ${PXR_PACKAGE} because PXR_ENABLE_GL_SUPPORT is OFF")
+        "Skipping ${PXR_PACKAGE} because PXR_ENABLE_GL_SUPPORT and PXR_ENABLE_METAL_SUPPORT is OFF")
     return()
 endif()
 
 if(APPLE)
     set(GARCH_GLPLATFORMCONTEXT       glPlatformContextDarwin)
     set(GARCH_GLPLATFORMDEBUGWINDOW   glPlatformDebugWindowDarwin)
     set(GARCH_SOURCE_EXTENSION        mm)
-    FIND_LIBRARY(APPKIT_LIBRARY AppKit)
+    if (NOT APPLEIOS)
+        list(APPEND optionalLibs      ${APPKIT_LIBRARY})
+    endif()
+    if (APPLEIOS)
+        FIND_LIBRARY(UIKIT_LIBRARY UIKit)
+    else()
+        FIND_LIBRARY(APPKIT_LIBRARY AppKit)
+    endif()
```

Comment:
```suggestion
    if (APPLEIOS)
        FIND_LIBRARY(UIKIT_LIBRARY UIKit)
    else()
        FIND_LIBRARY(APPKIT_LIBRARY AppKit)
        list(APPEND optionalLibs      ${APPKIT_LIBRARY})
    endif()
```

Reply:
Included a modification in the PR on Pixar's dev